pkgs/by-name/ma/material-cursors/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib, inkscape, xcursorgen, stdenvNoCC, fetchFromGitHub }:
+
+stdenvNoCC.mkDerivation {
+  pname   = "material-cursors";
+  version = "unstable-2023-11-30";
+
+  src = fetchFromGitHub {
+    owner = "varlesh";
+    repo = "material-cursors";
+    rev = "2a5f302fefe04678c421473bed636b4d87774b4a";
+    hash = "sha256-uC2qx3jF4d2tGLPnXEpogm0vyC053MvDVVdVXX8AZ60=";
+  };
+
+  nativeBuildInputs = [
+    inkscape
+    xcursorgen
+  ];
+
+  buildPhase = ''
+    runHook preBuild
+
+    # Silences the inkscape warning spam.
+    HOME=$(pwd) bash build.sh 2> /dev/null
+
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/icons
+    cp -r dist/* $out/share/icons/
+
+    runHook postInstall
+  '';
```

Comment:
```suggestion
  installFlags = [ "PREFIX=$(out)" ];
```

Reply:
wouldn't this error since the script uses bash?